src/examples/zap-baseline.py

```diff
@@ -349,6 +355,9 @@ def main(argv):
       logging.debug ('Ajax Spider complete')
 
     # Wait for passive scanning to complete
+    while (datetime.now() - start).seconds < ((mins * 60) + 10):
```

Comment:
I think I get it now :) So, by the time the spider has finished the passive scanner already finished too(?) and that does not give time to the external software to start?